src/System.Drawing.Graphics/src/System/Drawing/Graphics/Image.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
```

Comment:
You should add the licensing comment we use in this repo. See the DLLImports file I changed yesterday.